bindings/gumjs/gumdukmodule.c

```diff
@@ -50,6 +50,7 @@ GUMJS_DECLARE_FUNCTION (gumjs_module_map_find)
 GUMJS_DECLARE_FUNCTION (gumjs_module_map_find_name)
 GUMJS_DECLARE_FUNCTION (gumjs_module_map_find_path)
 GUMJS_DECLARE_FUNCTION (gumjs_module_map_update)
+GUMJS_DECLARE_FUNCTION (gumjs_module_map_copy_values)
```

Comment:
This function name is nice and descriptive, but for consistency we should name it `gumjs_module_map_values`.

Reply:
Oops, great point! Let's keep it like this 👍